src/status_im/chat/events/input.cljs

```diff
@@ -20,474 +23,566 @@
+
+;; TODO: remove this handler and leave only helper fn once all invocations are refactored
 (register-handler-db
- :clear-seq-arguments
- [trim-v]
- (fn [{:keys [current-chat-id chats] :as db} [chat-id]]
-   (let [chat-id (or chat-id current-chat-id)]
-     (-> db
-         (assoc-in [:chats chat-id :seq-arguments] [])
-         (assoc-in [:chats chat-id :seq-argument-input-text] nil)))))
+  :clear-seq-arguments
+  (fn [db]
+    (clear-seq-arguments db)))
 
 (register-handler-db
- ::update-seq-arguments
- [trim-v]
- (fn [{:keys [current-chat-id chats] :as db} [chat-id]]
-   (let [chat-id (or chat-id current-chat-id)
-         text    (get-in chats [chat-id :seq-argument-input-text])]
-     (-> db
-         (update-in [:chats chat-id :seq-arguments] #(into [] (conj % text)))
-         (assoc-in [:chats chat-id :seq-argument-input-text] nil)))))
+  ::update-seq-arguments
+  [trim-v]
+  (fn [{:keys [current-chat-id chats] :as db} [chat-id]]
+    (let [chat-id (or chat-id current-chat-id)
+          text    (get-in chats [chat-id :seq-argument-input-text])]
+      (-> db
+          (update-in [:chats chat-id :seq-arguments] #(into [] (conj % text)))
+          (assoc-in [:chats chat-id :seq-argument-input-text] nil)))))
 
 (register-handler-fx
- :send-seq-argument
- [trim-v]
- (fn [{{:keys [current-chat-id chats] :as db} :db} [chat-id]]
-   (let [chat-id          (or chat-id current-chat-id)
-         text             (get-in chats [chat-id :seq-argument-input-text])
-         seq-arguments    (get-in chats [chat-id :seq-arguments])
-         command          (-> (input-model/selected-chat-command db chat-id)
-                              (assoc :args (into [] (conj seq-arguments text))))]
-     {:dispatch [::request-command-data
-                 {:content             command
-                  :chat-id             chat-id
-                  :jail-id             (or (get-in command [:command :bot]) chat-id)
-                  :data-type           :validator
-                  :event-after-creator (fn [_ jail-response]
-                                         [::proceed-validation
-                                          jail-response
-                                          [[::update-seq-arguments chat-id]
-                                           [:send-current-message]]])}]})))
+  :send-seq-argument
+  (fn [{{:keys [current-chat-id chats] :as db} :db} _]
+    (let [text             (get-in chats [current-chat-id :seq-argument-input-text])
+          seq-arguments    (get-in chats [current-chat-id :seq-arguments])
+          command          (-> (input-model/selected-chat-command db current-chat-id)
+                               (assoc :args (into [] (conj seq-arguments text))))]
+      (request-command-data db {:content             command
+                                :chat-id             current-chat-id
+                                :jail-id             (or (get-in command [:command :bot]) current-chat-id)
```

Comment:
we could use `(get-in command [:command :bot] current-chat-id)` here, though this is legacy code

Reply:
Good point, I removed it almost everywhere (passing of the `chat-id` as argument for events relevant to current input), bot forgot about this occasion, will fix it.